cpp/src/slice2cs/Gen.cpp

```diff
@@ -3950,7 +3955,7 @@ Slice::Gen::DispatcherVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
 
         writeTieOperations(p);
 
-        _out << sp << nl << "private " << name << opIntfName << "_ _ice_delegate;";
+        _out << sp << nl << "private " << name << opIntfName << "_? _ice_delegate;";
```

Comment:
Kept the Tie code as-is. I am in favor of removing Ties entirely in a follow-up PR. What do you think?